File: 2089. Find Target Indices After Sorting Array/solution.py
class Solution(object):
  def targetIndices(self, nums, target):
    """
    :type nums: List[int]
    :type target: int
    :rtype: List[int]
    """
    count = 0
    lessThanCount = 0
    
    for num in nums:
      if num == target:
        count += 1
      elif num < target:
        lessThanCount += 1
    
    result = []
    
    for amount in range(count):
      result.append(lessThanCount)
      lessThanCount += 1
    
    return result

File: 2089. Find Target Indices After Sorting Array/solution.cpp
class Solution {
public:
  vector<int> targetIndices(vector<int>& nums, int target) {
    int count = 0;
    int lessThanCount = 0;

    for (int index = 0; index < nums.size(); index++){
      int num = nums[index];
      if (num == target){
        count += 1;
      }
      else if (num < target){
        lessThanCount += 1;
      }
    }

    vector<int> result;

    for (int amount = 0; amount < count; amount++){
      result.push_back(lessThanCount);
      lessThanCount += 1;
    }

    return result;
  }
};